ballerina/tests/55_declarative_authentication_service.bal

```diff
@@ -0,0 +1,156 @@
+    connectionPassword: "avix123",
+    userSearchBase: "ou=Users,dc=avix,dc=lk",
+    userEntryObjectClass: "inetOrgPerson",
+    userNameAttribute: "uid",
+    userNameSearchFilter: "(&(objectClass=inetOrgPerson)(uid=?))",
+    userNameListFilter: "(objectClass=inetOrgPerson)",
+    groupSearchBase: ["ou=Groups,dc=avix,dc=lk"],
+    groupEntryObjectClass: "groupOfNames",
+    groupNameAttribute: "cn",
+    groupNameSearchFilter: "(&(objectClass=groupOfNames)(cn=?))",
+    groupNameListFilter: "(objectClass=groupOfNames)",
+    membershipAttribute: "member",
+    userRolesCacheEnabled: true,
+    connectionPoolingEnabled: false,
+    connectionTimeout: 5,
+    readTimeout: 60
+};
+
+JwtValidatorConfigWithScopes jwtAuthConfig55WithScopes = {
+    jwtValidatorConfig: jwtAuthConfig55,
+    scopes: "write"
+};
+
+JwtValidatorConfigWithScopes jwtAuthConfig55EmptyScope = {
+    jwtValidatorConfig: jwtAuthConfig55
+};
+
+OAuth2IntrospectionConfigWithScopes oauth2config55WithScopes = {
+    oauth2IntrospectionConfig: oauth2IntroConfig55,
+    scopes: "write"
+};
+
+OAuth2IntrospectionConfigWithScopes oauth2config55EmptyScope = {
+    oauth2IntrospectionConfig: oauth2IntroConfig55
+};
+
+LdapUserStoreConfigWithScopes ldapUserStoreconfig55WithScopes = {
+    ldapUserStoreConfig: ldapUserStoreConfig,
+    scopes: "developer"
+};
+
+LdapUserStoreConfigWithScopes ldapUserStoreconfig55EmptyScope = {
+    ldapUserStoreConfig: ldapUserStoreConfig
+};
+
+FileUserStoreConfigWithScopes fileUserStoreConfig55WithScopes = {
+    fileUserStoreConfig: {},
+    scopes: "write"
+};
+
+FileUserStoreConfigWithScopes fileUserStoreConfig55EmptyScope  = {
+    fileUserStoreConfig: {}
+};
+
+@ServiceConfig {
+    auth: [
+        jwtAuthConfig55WithScopes,
+        oauth2config55WithScopes,
+        // ldapUserStoreconfig55WithScopes,
+        fileUserStoreConfig55WithScopes
+    ]
+}
+@ServiceDescriptor {descriptor: ROOT_DESCRIPTOR_55, descMap: getDescriptorMap55()}
+service "helloWorld55" on ep55WithScopes {
+
+    remote function hello55BiDiWithCaller(HelloWorld55StringCaller caller,
+     stream<string, Error?> clientStream) returns error? {
+        record {|string value;|}|Error? result = clientStream.next();
+        result = clientStream.next();
+        check caller->sendString("Hello");
+        check caller->complete();
+    }
+
+    remote function hello55BiDiWithReturn(stream<string, Error?> clientStream) 
+    returns stream<string, Error?>|error? {
+        return clientStream;
+    }
+
+    remote function hello55UnaryWithCaller(HelloWorld55StringCaller caller, string value) returns error? {
+        check caller->sendString(value);
+        check caller->complete();
+    }
+
+    remote function hello55UnaryWithReturn(string value) returns string|error? {
+        return value;
+    }
+}
+
+@ServiceConfig {
+    auth: [
+        jwtAuthConfig55EmptyScope,
+        oauth2config55EmptyScope,
+        // ldapUserStoreconfig55EmptyScope,
```

Comment:
ditto

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/343/commits/5d5a9b59b1f48a15996c9bd25c05d1ebff7df100